src/TouristInfoCards.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="cards-layout">
+      <div className="card">
```

Comment:
It looks like you have three separate copies of the same code to make a card in this component - could you extract a `TouristInfoCard` component to avoid that duplication?